packages/math-input/src/components/keypad/__tests__/keypad-v2.cypress.ts

```diff
@@ -0,0 +1,61 @@
+import renderSingleKeypad from "../../../../../../testing/render-keypad-with-cypress";
+
+type StringDictionary = {[key: string]: string};
+type TabArray = StringDictionary[];
+const tabs: TabArray = [
+    {name: "Operators", specialButton: "EXP_2"},
+    {name: "Extras", specialButton: "PI"},
+
+    {name: "Geometry", specialButton: "COS"},
+    {name: "Numbers", specialButton: "NUM_7"},
+];
+
+describe("Keypad v2", () => {
+    it("renders", () => {
+        renderSingleKeypad((key) => {});
+    });
+
+    tabs.forEach((tab) => {
+        it(
+            "switches to the " +
+                tab["name"] +
+                " tab when we press on the " +
+                tab["name"] +
+                " button",
+            () => {
+                renderSingleKeypad((key) => {});
+
+                // currently clicking on the bottom left due to button re-rendering after mousedown but before mouseup (only in Cypress)
+                cy.get('[aria-label="' + tab["name"] + '"]').click(
```

Comment:
```suggestion
                cy.get('[aria-label="' + tab.name + '"]').click(
```
I typically recommend this kind of dot notation unless you're dynamically accessing properties on an object. (JavaScript nit)